### PR TITLE
chore(influx): improve usage for influx pkg commands

### LINKED ISSUE #18324: Extend influx CLI pkg commands with better usage

### PR DESCRIPTION
**Pull request by jsteenb2**
Closes #18324 

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass
- [x] Documentation updated or issue created (provide link to issue/pr)

### CODE REVIEW

File: cmd/influx/pkg.go [hunk 1]
```diff
@@ -333,6 +430,15 @@ func (b *cmdPkgBuilder) pkgExportAllRunEFn(cmd *cobra.Command, args []string) er
...
 	cmd := b.newCmd("stack $STACK_ID", b.pkgExportStackRunEFn, true)
 	cmd.Short = "Export all existing resources for an organization as a package"
+	cmd.Long = `
+	The pkg export stack command exports the resources associated with a stack as
+	they currently exist in the platform. All the same metadata.name fields will be
+	reused.
+
+	Example:
+		# Export by a stack
+		influx pkg export stack $STACK_ID
```
Comment (jsteenb2) [inline cmd/influx/pkg.go:440]: v2 docs haven't been added yet. Should be available come next release. Will backfill that when it is available. Perhaps we can do it in tandem @sanderson?

File: cmd/influx/pkg.go [hunk 2]
```diff
@@ -97,6 +97,50 @@ func (b *cmdPkgBuilder) cmd() *cobra.Command {
...
+
+	Examples:
+		# Apply a template via a file
+		influx pkg -f $PATH_TO_TEMPLATE/template.json
+
+		# Apply a stack that has associated templates. In this example the stack has a remote
+		# template associated with it.
+		influx pkg --stack-id $STACK_ID
+
+		# Apply a template associated with a stack. Stacks make tempalte application idempotent.
```
Comment (pierwill) [inline cmd/influx/pkg.go:115]: `template`

File: cmd/influx/pkg.go [hunk 3]
```diff
@@ -97,6 +97,50 @@ func (b *cmdPkgBuilder) cmd() *cobra.Command {
...
+			-f $PATH_TO_TEMPLATE/template_1.json \
+			-f $PATH_TO_TEMPLATE/template_2.yml
+
+		# Apply a template from a url
+		influx pkg -u https://raw.githubusercontent.com/influxdata/community-templates/master/docker/docker.yml
+
+		# Apply a template from STDIN
+		cat $TEMPLATE.json | influx pkg --encoding json
+
+		# Applying a directory of templates, takes everything from provided dir
```
Comment (pierwill) [inline cmd/influx/pkg.go:129]: Would expand `dir` to `directory` here and below.

**Comment by jsteenb2** [comment]
@pierwill thank you! pushed up changes per your feedback 👍

File: cmd/influx/pkg.go (hunk 1 above)
Comment (sanderson) [reply cmd/influx/pkg.go:440]: @jsteenb2 This should be pretty easy to add. I'll do it this afternoon. The task-based stack management docs are a different PR and will get finalized once I wrap up my InfluxDays presentation.

Comment (jsteenb2) [reply cmd/influx/pkg.go:440]: awesome, sounds great 👍

Comment (sanderson) [reply cmd/influx/pkg.go:440]: https://github.com/influxdata/docs-v2/pull/1083

**Pull request merged**